resources/js/components/ui/form/Checkbox.tsx

```diff
@@ -0,0 +1,58 @@
+import {Component, JSX, createEffect, splitProps} from "solid-js";
+import * as checkbox from "@zag-js/checkbox";
+import {normalizeProps, useMachine} from "@zag-js/solid";
+import {createMemo, createUniqueId} from "solid-js";
+import s from "./Checkbox.module.scss";
+import {ValidationMessages} from "./ValidationMessages";
+import {useFieldLabel} from "./FieldLabel";
+
+export interface CheckboxProps extends Omit<JSX.InputHTMLAttributes<HTMLInputElement>, "onChange"> {
+  name: string;
+  label?: string;
+  disabled?: boolean;
```

Comment:
A tego nie ma w tym typie z którego dziedziczysz te propsy przypadkiem? Bo te TextFieldy to nie biorą disabled explicite tylko właśnie dziedzicząc.